.gitignore

```diff
@@ -117,7 +117,7 @@ mkdocs.yml
 /docs/refmans/gui/viselements/generic/charts/*.md
 /docs/refmans/gui/viselements/corelements/*.md
 !/docs/userman/gui/gui_example.ipynb
-/docs/xrefs
+/docs/userman/xrefs
```

Comment:
xrefs should have been moved from `docs/userman` to `docs` directly.

Reply:
But you're absolutely right!
Thx